### ISSUE TITLE
Erroneous data for countries that span continental boundaries

### ISSUE THREAD

**Issue by brandondrew** [countries/countries#780]
Continental boundaries are not as clear as we might like.   If we accept the conventional division between Europe and Asia, then both Russia and Turkey cross the boundary, with by far the greater part of their land in Asia.

And yet this gem lists both of them as being in Europe.  

```
require 'countries/global'
turkey = Country['TR']
russia = Country['RU']
turkey.continent # => "Europe"
russia.continent # => "Europe"
```

For Russia this _might_ make sense, since the bulk of the population (if I'm not mistaken) is in Europe, but for Turkey this is most certainly wrong.  The vast majority of the Turkish people are very definitely in Asia, and the portion of the country's land that is in Europe is absolutely tiny.

> East Thrace or Eastern Thrace, also known as Turkish Thrace or European Turkey, is the part of Turkey that is geographically a part of Southeast Europe. It accounts for 3.4% of Turkey's land area but comprises 15% of its total population.
-- https://en.wikipedia.org/wiki/East_Thrace

Ideally I think it makes more sense to have a `:continents` key which returns an array. For backward compatibility it would make sense to keep `:continent`, but it should show the _primary_ continent, preferably by some criteria that depends on measurable data rather than someone's preference.

I'd be happy to submit PRs for correcting the `:continent` to reflect primary continent if there's an openness to it.  If so, the main question would be do we want to use land area or population to determine which continent is primary?

The other one is a bit more complicated, and therefore might perhaps make sense to do in stages, as we would run into questions like "Is French Guiana actually a part of France?" (I'm told it is by a geographic expert I trust, but it may not be as clear-cut as Russia and Turkey.)

**Comment by pmor** [comment]
Yeah, I think you have a good idea there, I'll think about this a bit more and try to come up with something
